### PR TITLE
Update params to standard

### LINKED ISSUE #12: Ensure that span tracers follow guidelines
There are OpenTelemetry guidelines for attributes and span names in place, we should look into following them: [Trace Guidelines](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/semantic_conventions/README.md)

### PR DESCRIPTION
**Pull request by maciej-szlosarczyk**
Fixes #12, at least for the places where it is possible and can be done.

### CODE REVIEW

**Comment by sauron-droid** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please ask for approval from **maciej-szlosarczyk** after the PR has been reviewed.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=mindvalley%2Fmv-opentelemetry).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/mindvalley/mv-opentelemetry/blob/main/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Review by feliperenan — APPROVED** [review]
LGTM 👍🏻

**Comment by sauron-droid** [comment]
LGTM label has been added.  <details>Git tree hash: 2faf962eaa54324e592007f59d0a8082f477a2ed</details>

**Comment by sauron-droid** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/mindvalley/mv-opentelemetry/pull/18#pullrequestreview-849276624" title="Approved">feliperenan</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=mindvalley%2Fmv-opentelemetry).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/mindvalley/mv-opentelemetry/blob/main/OWNERS)~~ [feliperenan]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: lib/mv_opentelemetry/ecto.ex
Comment (antonsatin) [inline lib/mv_opentelemetry/ecto.ex:28]: You use it just once, i'd inline it

File: lib/mv_opentelemetry/ecto.ex
Comment (antonsatin) [inline lib/mv_opentelemetry/ecto.ex:75]: Why do you first match on an error case rather than successful one?

File: lib/mv_opentelemetry/ecto.ex
Comment (antonsatin) [inline lib/mv_opentelemetry/ecto.ex:91]: This one i'm not entirely sure - can't error be something rather than exception? 
- If it can then why its ignored and not put into status
- If it cannot then why its ignored and not reported anywhere as an actual error

File: lib/mv_opentelemetry/ecto.ex
Comment (maciej-szlosarczyk) [reply lib/mv_opentelemetry/ecto.ex:73]: Because the successful case doesn't matter - the value is not re-used elsewhere.

OpenTelemetry.Span.set_status/3 is a side-effectful function that manipulates data in ets.

**Comment by sauron-droid** [comment]
New changes are detected. LGTM label has been removed.

**Comment by sauron-droid** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please ask for approval from **feliperenan** after the PR has been reviewed.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=mindvalley%2Fmv-opentelemetry).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/mindvalley/mv-opentelemetry/blob/main/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by sauron-droid** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please ask for approval from **feliperenan** after the PR has been reviewed.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=mindvalley%2Fmv-opentelemetry).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/mindvalley/mv-opentelemetry/blob/main/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: lib/mv_opentelemetry/ecto.ex
Comment (antonsatin) [reply lib/mv_opentelemetry/ecto.ex:89]: Hmm, maybe `inspect(error)`? It will always give a string and it has more info than empty string anyway

Comment (maciej-szlosarczyk) [reply lib/mv_opentelemetry/ecto.ex:89]: That is not a good idea for performance reasons - We cannot know if it is a 4-levels nested 32 field map, a very long list or something entirely different.

Comment (antonsatin) [reply lib/mv_opentelemetry/ecto.ex:89]: Ok!

**Review by antonsatin — APPROVED** [review]

**Comment by sauron-droid** [comment]
LGTM label has been added.  <details>Git tree hash: d264d215fc554f18bf37f6b434e349a71ac7492a</details>

**Comment by sauron-droid** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/mindvalley/mv-opentelemetry/pull/18#pullrequestreview-850119527" title="Approved">antonsatin</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=mindvalley%2Fmv-opentelemetry).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/mindvalley/mv-opentelemetry/blob/main/OWNERS)~~ [antonsatin]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**
